source/includes/usage-examples/code-snippets/Command.java

```diff
@@ -0,0 +1,34 @@
+package usage.examples;
+
+import org.bson.BsonDocument;
+import org.bson.BsonInt64;
+import org.bson.Document;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoException;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
+
+
+public class RunCommand {
+    public static void main(String[] args) {
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+        try (MongoClient mongoClient = MongoClients.create(uri)) {
+
+            MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+
+            try {
+                // find the storage statistics for the "sample_mflix" database using the 'dbStats' command
```

Comment:
Nitpick: Other than the URI replacement, let's avoid using comments in the code since our driver docs user studies showed that readers are more likely to read text than comments in code. I think I've followed this in the other usage examples, but if you find an instance, I'll fix it.